ncdump/CMakeLists.txt

```diff
@@ -213,6 +213,7 @@ endif()
   add_sh_test(ncdump tst_64bit)
   add_sh_test(ncdump tst_lengths)
   add_sh_test(ncdump tst_calendars)
+  add_sh_test(ncdump tst_calendars_nc4)
```

Comment:
I believe I placed this new test in the wrong section. I see the `IF (USE_HDF5)` section below this and I suspect that is where it belongs. I'll try moving it.